File: gluecodium/src/test/resources/smoke/typedefs/output/android/com/example/smoke/TypeCollection.java
/*
 *
 */
package com.example.smoke;
public final class TypeCollection {
    public static final long INVALID_STORAGE_ID = 0L;
    public static final class Point {
        public double x;
        public double y;
        public Point(final double x, final double y) {
            this.x = x;
            this.y = y;
        }
    }
    public static final class StructHavingAliasFieldDefinedBelow {
        public long field;
        public StructHavingAliasFieldDefinedBelow(final long field) {
            this.field = field;
        }
    }
}

File: gluecodium/src/test/resources/smoke/typedefs/output/android-kotlin/com/example/smoke/TypeCollection.kt
/*

 *
 */

@file:JvmName("TypeCollection")

package com.example.smoke


class TypeCollection {

    class Point {
        var x: Double
        var y: Double



        constructor(x: Double, y: Double) {
            this.x = x
            this.y = y
        }





    }

    class StructHavingAliasFieldDefinedBelow {
        var field: Long



        constructor(field: Long) {
            this.field = field
        }





    }








    companion object {
        val INVALID_STORAGE_ID: Long = 0L
    }
}

